service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -520,11 +520,15 @@ public boolean hasAllocatedJudgeEmail() {
     }
 
     @JsonIgnore
-    public HearingBooking getMostUrgentHearingBookingAfter(LocalDateTime time) {
+    public Optional<HearingBooking> getNextHearingAfter(LocalDateTime time) {
```

Comment:
maybe adding unit test on those 2 method in getMostUrgentHearingBookingAfter and getNextHearingAfter  caseDataTest?

Reply:
added some unit tests 👍